invisible_cities/detsim/sensor_utils_test.py

```diff
@@ -115,3 +116,15 @@ def test_create_timestamp_greater_with_greater_eventnumber():
     assert abs(timestamp_1) == timestamp_1
     assert timestamp_1      <  timestamp_2
     assert abs(timestamp_2) == timestamp_2
+
+
+@mark.filterwarnings("ignore:Zero rate")
+def test_get_rate_safe_allways_possitive():
+    """
+    Be sure that it woll always return the rate value such that
```

Comment:
Typo (woll -> will). I would rewritte the docstring to something like "Check the rate is always positive/physical".

On the other hand, since you are testing a single value whose output is determined (hardcoded in the function being tested) I think it is better to check the exact result.